pkgs/_cfe_macros/lib/macro_implementation.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:_macro_host/macro_host.dart';
+import 'package:dart_model/dart_model.dart';
+// ignore: implementation_imports
+import 'package:front_end/src/macros/macro_injected_impl.dart' as injected;
+import 'package:macro_service/macro_service.dart';
+import 'package:macros/macros.dart';
+// ignore: implementation_imports
+import 'package:macros/src/executor.dart' as injected;
+
+/// Injected macro implementation for the analyzer.
+class MacroImplementation implements injected.MacroImplementation {
```

Comment:
nit: I would rename this one also (`CfeMacroImplementation`), and rename the analyzer one also.

Reply:
Done.